manifests/input.pp

```diff
@@ -18,6 +18,7 @@
   $options        = undef,
   $single_section = undef,
   $sections       = undef,
+  $tags           = undef,
```

Comment:
manifests/input.pp has some documentation within a comment block at the top describing all of the arguments that can be passed to the class. Can you add a comment for $tags that mentions what its about & what data type should be passed in?

Reply:
Updated and pushed